whois-api/src/test/java/net/ripe/db/whois/api/elasticsearch/AbstractElasticSearchIntegrationTest.java

```diff
@@ -0,0 +1,92 @@
+package net.ripe.db.whois.api.elasticsearch;
+
+import com.google.common.util.concurrent.Uninterruptibles;
+import net.ripe.db.whois.api.AbstractIntegrationTest;
+import net.ripe.db.whois.api.ElasticSearchHelper;
+import net.ripe.db.whois.common.elasticsearch.ElasticIndexService;
+import org.apache.commons.lang3.StringUtils;
+import org.elasticsearch.client.RequestOptions;
+import org.elasticsearch.index.query.QueryBuilders;
+import org.elasticsearch.index.reindex.DeleteByQueryRequest;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.testcontainers.elasticsearch.ElasticsearchContainer;
+
+import java.io.IOException;
+import java.util.concurrent.TimeUnit;
+
+public abstract class AbstractElasticSearchIntegrationTest extends AbstractIntegrationTest {
+
+    public static final String ENV_DISABLE_TEST_CONTAIENRS = "test.containers.disabled";
+    private static ElasticsearchContainer elasticsearchContainer;
+
+    @Autowired
+    ElasticIndexService elasticIndexService;
+
+    @Autowired
+    ElasticFullTextIndex elasticFullTextIndex;
+
+    @BeforeAll
+    public static void setUpElasticCluster() {
+        if (StringUtils.isBlank(System.getProperty(ENV_DISABLE_TEST_CONTAIENRS))) {
+            if (elasticsearchContainer == null) {
+                elasticsearchContainer = new ElasticsearchContainer("docker.elastic.co/elasticsearch/elasticsearch:7.15.0");
+                elasticsearchContainer.start();
+            }
+
+            System.setProperty("elastic.host", elasticsearchContainer.getHttpHostAddress().split(":")[0]);
```

Comment:
Is this logic IPv6 specific? Can you separate the split logic into a separate method?